elastalert/config.py

```diff
@@ -372,7 +372,7 @@ def create_alert(alert, alert_config):
             alert_field = [alert_field]
 
         alert_field = [normalize_config(x) for x in alert_field]
-        alert_field = sorted(alert_field, key=lambda (a, b): alerts_order.get(a, -1))
+        alert_field = sorted(alert_field, key=lambda (a, b): alerts_order.get(a, 1))
```

Comment:
This is probably fine, unless you want to ensure that email always immediately follows JIRA, in which case you should set the default to something higher, like `alerts_order.get(a,2)`

Reply:
why do we need alert order map? `rule['alert']` is [always list](https://github.com/Yelp/elastalert/blob/5cdf3b3e409016c14e5bf76a0203729e9795fd5a/elastalert/config.py#L372) so valid order is already defined in rule config and pipelining should also work, just place `email` below `jira`
```yaml
alert:
   - "jira"
   - "email"

Reply:
So that people can take advantage of the alert pipeline without having to explicitly order the alerts correctly. Anything that adds something (like JIRA) can default to be run first.